src/maps/map.py

```diff
@@ -84,13 +85,14 @@ def add_meeting_places_to_map(self, sections: pd.DataFrame, colour, marker_data:
         if sections.empty:
             return
 
-        if not self.map_plotter.layers.get(layer):
-            self.map_plotter.add_layer(layer, cluster_markers)
+        if not self._map_plotter.layers.get(layer["name"]):
+            layer = dict(name="Sections", markers_clustered=False) if layer is None else layer
```

Comment:
Do you think it would be helpful to add the default value to the docstring as it's not in the function definition? (P.S. the docstring will need updating from layer str to layer dict)

Reply:
Done both!